build.gradle

```diff
@@ -2,8 +2,8 @@ apply plugin: 'com.github.ben-manes.versions'
 
 buildscript {
   ext.versions = [
-      'minSdk': 9,
-      'compileSdk': 28,
+      'minSdk': 14,
```

Comment:
curious: why not leave as minSdk 9?

Reply:
Something required minSdk 14. I don't recall what it was.

Reply:
looks like it was viewbinding:

```
* What went wrong:
Execution failed for task ':timber-sample:processDebugManifest'.
> Manifest merger failed : uses-sdk:minSdkVersion 9 cannot be smaller than version 14 declared in library [androidx.databinding:viewbinding:4.0.0] /Users/jrod/.gradle/caches/transforms-2/files-2.1/9d55cbb08326255dbf4b969fbb520979/viewbinding-4.0.0/AndroidManifest.xml as the library might be using APIs not available in 9
        Suggestion: use a compatible library with a minSdk of at most 9,
                or increase this project's minSdk version to at least 14,
                or use tools:overrideLibrary="androidx.viewbinding" to force usage (may lead to runtime failures)
```

Reply:
Is it worth to use `findViewById` and roll this back to minSdk 9?

Reply:
Or at least specify minSdk 9 for the library and 14 for the sample?

Reply:
I think it's fine to bump to min 14 unless @JakeWharton has concerns